geometry_msgs/PoseStamped gripper_pose
geometry_msgs/PoseStamped object_pose
int32 observation
